### ISSUE TITLE
Crash on enum declaration

### ISSUE THREAD

**Issue by Graham42** [thorbenvh8/prettier-java#53]
Sample code
```java
public class MyTest {
    public static enum VALID_THINGS {
        FIRST,
        SECOND
    }
}
```

Stacktrace:
```
[error] MyTest.java: TypeError: Cannot read property 'declarations' of undefined
[error]     at printEnumDeclaration (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/printer.js:528:17)
[error]     at printNode (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/printer.js:2237:14)
[error]     at Object.genericPrint [as print] (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/printer.js:2631:10)
[error]     at genericPrint$6 (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23998:18)
[error]     at comments.printComments.p (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23936:14)
[error]     at Object.printComments (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:8252:19)
[error]     at printGenerically (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23934:22)
[error]     at FastPath.call (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:21082:18)
[error]     at printClassBodyMemberDeclaration (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/printer.js:368:18)
[error]     at printNode (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/printer.js:2225:14)
[error]     at Object.genericPrint [as print] (/home/grahamm/<snipped-project-dir>/node_modules/prettier-plugin-java/src/printer.js:2631:10)
[error]     at genericPrint$6 (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23998:18)
[error]     at comments.printComments.p (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23936:14)
[error]     at Object.printComments (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:8252:19)
[error]     at printGenerically (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:23934:22)
[error]     at FastPath.map (/home/grahamm/<snipped-project-dir>/node_modules/prettier/bin-prettier.js:21136:19)
```

**Comment by thorbenvh8** [comment]
Fixed in next release :)

**Comment by thorbenvh8** [comment]
Fixed in 0.0.10!
Thanks for the raising the issue! 👍
